resources/nb-meta-check.py

```diff
@@ -26,6 +27,14 @@ def error(msg):
     if 'description' not in meta:
         error(f'No `description` in `meta` section of {f}')
 
+    # Authors must be a non-empty list
+    if (
+        'authors' not in meta
+        or not isinstance(meta['authors'], list)
```

Comment:
Does the design allow for multiple authors? If not this should be a singular entry

Reply:
It does, in those cases we use the letter and title of the first author

![image](https://github.com/singlestore-labs/spaces-notebooks/assets/22195590/c45e84b5-fe43-4bc9-8595-84e380e7f428)